### ISSUE TITLE
A11y_Azure App Service extension for VS Code_Files_Popup_ScreenReader: The screen reader announces the Saving "hostingstart.html" notification pop up as "Notification list list".

### ISSUE THREAD

**Issue by v-gitsay** [microsoft/vscode-azureappservice#2136]
### Actual:
NVDA announces the Saving "hostingstart.html" notification pop up as "Notification list list".

**Note**: 

1. The issue is observed with JAWS.
2. The issue is observed with all the Notification Pop up that appear for "Initializing SSH" pop up scenario.

### Expected:
NVDA should announce the Saving "hostingstart.html" notification pop up as "Notification Saving hostingstart.html".

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev2;#NVDA;#JAWS;#SH_AzureAppServiceextensionforVSCode_Win32_Sep2021;#Azure Tools for VS Code;#DesktopApp;#WCAG1.3.1;#AINotInScope;#FTP;#ScreenReader;#Element:Popup;#Linked:Bug135638;#External:VSCode;
### Environment Details:
Application Name: Azure App Service extension for VS Code
Screen Reader: NVDA/JAWS
Operating system: Windows 10

### Repro Steps:

1. Launch VS Code application.
2. Tab till "Azure" and press enter.
3. Tab till "app service " dropdown and press enter.
4. Tab till "Files" dropdown and press enter.
5. Tab to any file and press enter.
6. Edit by adding "make changes" and save(ctrl + S) the "SRC" file..
7. Tab till the Pop up Saving "hostingstart.html".
8. Check if the NVDA is announcing Pop up details correctly.

### User Impact:
Screen reader users will not know that which notification pop up is on focus is the NVDA only announces "Notification list list".

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Please find the below attachment for reference:**
![2136](https://user-images.githubusercontent.com/86710911/134297742-426c50b6-3277-4d1c-927c-56416326a43d.gif)

**Comment by nturinski** [comment]
Using [this](https://code.visualstudio.com/api/get-started/your-first-extension) to create a basic extension , I am able to reproduce this issue.

Here is my output from the screen reader:
![image](https://user-images.githubusercontent.com/5290572/135365177-0325d0f8-0a6f-4cb5-bc7d-24ab60ff4fa7.png)


An extension cannot control how VS Code interacts with the screen reader. This issue should be filed on VS Code: https://github.com/microsoft/vscode/issues

**Comment by kupatkar99** [comment]
Issue is fixed
![A11y_Azure App service extensions](https://user-images.githubusercontent.com/98530069/166912612-94d25697-7436-4aa6-9794-17910a2c6c8e.gif)

**Comment by kupatkar99** [comment]
Issue is fixed.
![Azure-service-extension](https://user-images.githubusercontent.com/98530069/166932937-c2b4f8d8-75a3-4353-93e4-967cca02fd2b.gif)

**Comment by nturinski** [comment]
@kupatkar99 Are we okay to close this issue then?

**Comment by v-gitsay** [comment]
As per above attachment by @kupatkar99 issue is fixed hence closing the bug
